datadog/agent/install.go

```diff
@@ -68,6 +69,20 @@ func NewInstaller(vm *vm.UnixLikeVM, options ...func(*params) error) (*Installer
 	return &Installer{dependsOn: lastCommand}, err
 }
 
+func addTelemetry(options []func(*params) error) []func(*params) error {
+	config := `
+instances:
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
You can embed this - advantage is yaml formatting.

Reply:
I think the inline content is small enough to be kept in the code (for a better locality).